goalie/mesh_seq.py

```diff
@@ -234,8 +235,10 @@ def _function_spaces_consistent(self) -> bool:
             )
         return consistent
 
-    @property
-    def function_spaces(self) -> list:
+    def update_function_spaces(self) -> list:
```

Comment:
@jwallwork23 - I think your edits here make more sense than my interpretation of your suggestions and it is a better descriptive name of the method you had already implemented.